.storybook/preview.js

```diff
@@ -18,50 +18,50 @@ window.process = window.process || {};
 window.process.env = window.process.env || {};
 window.process.env.NODE_ENV = window.process.env.NODE_ENV || 'production';
 
+addDecorator(withA11y);
```

Comment:
This code previously living inside of the async function below meant we were relying on a trick of load timing to ensure that it was available for the first story to be visited. These changes were pointed out by the maintainer of `@open-wc/demoing-storybook` as the best way to address this situation in #492